src/map.cpp

```diff
@@ -4177,6 +4177,12 @@ item &map::add_item_at( const tripoint &p,
     submap *const current_submap = get_submap_at( p, lx, ly );
     current_submap->is_uniform = false;
 
+    if( new_item.is_map() && !new_item.has_var( "reveal_map_center_omt" ) ) {
+        //new_item.set_var( "origin_abs", g->m.getabs( p ) );
```

Comment:
Looks like you forgot to remove some commented out code.

Reply:
Thanks.